test/integration/cdi/example-driver/app/server.go

```diff
@@ -77,6 +77,7 @@ func NewCommand() *cobra.Command {
 
 	fs = sharedFlagSets.FlagSet("CDI")
 	driverName := fs.String("drivername", "example-driver.cdi.k8s.io", "Resource driver name.")
+	cdiDir := fs.String("cdi-dir", "/var/run/cdi", "CDI directory where the driver sockets and CDI files will be")
```

Comment:
"/var/run/cdi" should be a constant.

Reply:
Umm, sorry??
What you said before:
> get CDI directory from the command line with /var/run/cdi as a default

What @pohly said right below your comment:
> with <cdi directory being configurable and defaulting to /var/run/cdi

In go language, is there a way to make it constant after getting the value from the user...?

Reply:
I'm talking about "/var/run/cdi", not about cdiDir.

Reply:
please use this constant: https://github.com/container-orchestrated-devices/container-device-interface/blob/master/pkg/cdi/spec-dirs.go#L30

Reply:
So, not configurable?

Reply:
Ah, I got it, now. Thanks!! :)

Reply:
not resolved, but marked as resolved again!